packages/renderer/src/lib/onboarding/Onboarding.svelte

```diff
@@ -1,3 +1,25 @@
+<style lang="postcss">
+#stepBody::-webkit-scrollbar {
```

Comment:
Can't we have this extracted ?

Reply:
I gave a look but it seems not possible :/ 
The code related to postcss needs to be here.

My knowledge about preprocessing is not that vast, i was expecting that if i created a file with extension `.pcss` it would trigger postcss but it does not work apparently. I cannot create a simple .css file.

If someone has any suggestions i'm all ears.